target/linux/rockchip/armv8/base-files/etc/board.d/02_network

```diff
@@ -17,6 +17,7 @@ rockchip_setup_interfaces()
 	xunlong,orangepi-r1-plus-lts)
 		ucidef_set_interfaces_lan_wan 'eth1' 'eth0'
 		;;
+	friendlyarm,nanopc-t6|\
```

Comment:
please put this section above `friendlyarm,nanopi-r2c`

Reply:
fixed